### PR TITLE
[OYPD-165] Add Image and video embed and image attach tests.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-165
D.o https://www.drupal.org/node/2865819

JS dependent tests added.
* Add test image embed on landing page. 
* Add test video embed on landing page.
* Add test document embed on landing page.
* Add test image attach to blog field.

Steps to review:
- [x] New tests passed

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build874 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build874 |
| Installation process | http://install.openy-dev.ffwua.com/build874/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/874/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build874/profiles/contrib/openy/build/reports/behat

**Comment by alexschedrov** [comment]
1. Document functinality is not covered yet.

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build881 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build881 |
| Installation process | http://install.openy-dev.ffwua.com/build881/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/881/

**Comment by Kerby70** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build888 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build888 |
| Installation process | http://install.openy-dev.ffwua.com/build888/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/888/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build888/profiles/contrib/openy/build/reports/behat

**Comment by alexschedrov** [comment]
@Kerby70 any updates/issues with documents?

**Comment by Kerby70** [comment]
@Sanchiz Sorry I had committed locally and my push to github had failed without my noticing.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build893 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build893 |
| Installation process | http://install.openy-dev.ffwua.com/build893/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/893/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build893/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

File: tests/features/content_types/blog_post.feature [hunk 1]
```diff
@@ -17,13 +21,61 @@ Feature: Blog Content type
...
-    And I fill in "Title" with "From OpenY Automation Blogger"
+    And I fill in "Title" with "Behat test blog"
     And I select "BEHAT BRANCH 01" from "Location"
-    And I fill in the following:
-      | Category    | BEHAT CATEGORY ONE                          |
-      | Description | This could be a draft for a wonderful post. |
+    And I fill in "Category" with "BEHAT CATEGORY ONE"
+    When I click "//*[@id='edit-group-content-area']/summary" xpath element
+    And I scroll to "#cke_edit-field-blog-description-0-value" element
+    And I should see an "#cke_edit-field-blog-description-0-value" element
```
Comment (AlexNetman) [inline tests/features/content_types/blog_post.feature:32]: Seems we don't need that line. Cause it is always true, if previous one is done. Am I correct?

**Pull request merged**